jax/random.py

```diff
@@ -418,6 +419,43 @@ def _normal(key, shape, dtype):
   return onp.array(onp.sqrt(2), dtype) * lax.erf_inv(u)
 
 
+def multivariate_normal(key, mean=0.0, cov=1.0, shape=(), dtype=onp.float64):
+  dtype = xla_bridge.canonicalize_dtype(dtype)
+  return _multivariate_normal(key, mean, cov, shape, dtype)
+
+@partial(jit, static_argnums=(3, 4))
+def _multivariate_normal(key, mean, cov, shape, dtype):
+  """Sample multivariate normal random values with given shape, mean, and covariance.
+
+  Args:
+    key: a PRNGKey used as the random key.
+    mean: optional, a scalar or array of mean values along each dimension
+    cov: optional, a scalar (isotropic), vector (diagonal covariance matrix), or full covariance matrix
+    shape: optional, a tuple of nonnegative integers representing the shape.
+
+  Returns:
+    A random array with latent dimension of (max(asarray(mean).ndim, asarray(cov).ndim)),)
+  """
+  _check_shape("multivariate_normal", shape)
+  if hasattr(mean, "shape") and mean.ndim > 1:
+        raise ValueError("Mean cannot have more than 1 dimension.")
```

Comment:
Indentation is a little wonky in these if/else blocks?

Reply:
I updated the indentation to consistently match the 2 space convention used in Jax. The build failed for Python versions 3.6 and 3.7 with what appears to be an unrelated error:

```
RemoveError: 'setuptools' is a dependency of conda and cannot be removed from
conda's operating environment.
The command "conda update --yes conda" failed and exited with 1 during .
Your build has been stopped.
```